packages/expect/src/index.ts

```diff
@@ -53,18 +52,18 @@ class JestAssertionError extends Error {
   matcherResult: any;
 }
 
-const isPromise = obj =>
+const isPromise = (obj: any) =>
```

Comment:
```suggestion
const isPromise = (obj: unknown) =>
```

Reply:
This doesnt work since .then doesnt exist on type object

Reply:
could even go `const isPromise = (obj: any): obj is PromiseLike => {` then the calling code could type the true condition.

Reply:
Yup! I've gone with that typeing